httpaf-lwt.opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+name: "httpaf-lwt"
+maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+authors: [ "Anton Bachin <antonbachin@yahoo.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/inhabitedtype/httpaf"
+bug-reports: "https://github.com/inhabitedtype/httpaf/issues"
+dev-repo: "git+https://github.com/inhabitedtype/httpaf.git"
+build: [
+  ["jbuilder" "subst" "-p" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "angstrom-lwt-unix"
```

Comment:
is `angstrom-lwt-unix` actually being used? I don't see it being referenced in the `jbuild` file

Reply:
Good catch, removed the dep.